### DISCUSSION TITLE
issue while installing cQube locally

### DISCUSSION THREAD

**Discussion by priya23595** [Sunbird-cQube/community#805] [Q&A]
When am installing the cQube locally facing with this issue please help me resolve this
![Screenshot from 2023-07-13 17-27-04](https://github.com/Sunbird-cQube/community/assets/136596601/f907d670-9785-473a-b941-7c99089d6733)
![Screenshot from 2023-07-14 11-28-19](https://github.com/Sunbird-cQube/community/assets/136596601/d328660c-2ccf-4a71-b2ec-829e8b849111)

**Comment by RNarmatha** [comment]
@asifbasha61 - Please look into this issue

**Comment by priya23595** [comment]
Hello
Can u share the mail_id of Asif sir and Rajesh sir so that i can mail

**Comment by RNarmatha** [comment]
Asif-asif.abdul@tibilsolutions.com
Rajesh-rajesh.bommala@tibilsolutions.com

**Comment by priya23595** [comment]
can we connect??to resolve this issue

**Comment by RNarmatha** [comment]
@priya23595 - our team members working on the P0 items. I will check with the team availability and get back to you .Thank you

**Comment by RNarmatha** [comment]
https://meet.google.com/hip-qaim-zss
Please join the call

**Comment by priya23595** [comment]
hello mam i am busy with other meeting can we connect now?

**Comment by RNarmatha** [comment]
Hi Priyanka,

can we connect in the same link

**Comment by priya23595** [comment]
yes

**Comment by priya23595** [comment]
Hi
when we connect can u check with asif and update me

**Comment by RNarmatha** [comment]
Asif- Can you please give an reply

**Comment by priya23595** [comment]
any updates? can u share the free time of Asif sir in which we will connect

**Reply by asifbasha61** [reply]
Hi @priya23595 We can connect now.

**Comment by priya23595** [comment]
yes

**Comment by priya23595** [comment]
can u share the meet link

**Comment by priya23595** [comment]
can u please join 
https://meet.google.com/yki-knqj-tqj

**Reply by asifbasha61** [reply]
let me in

**Comment by priya23595** [comment]
Thank you so much Asif sir and Narmatha mam for ur support 
It's Done with locally injection and data visualisation part

**Reply by asifbasha61** [reply]
Thanks Priyanka.

**Comment by RNarmatha** [comment]
Thank you for the updates Priyanka

**Reply by asifbasha61** [reply]
Narmatha, We can close this thread.
